geodiff/src/drivers/sqlitedriver.cpp

```diff
@@ -222,7 +222,7 @@ std::vector<std::string> SqliteDriver::listTables( bool useModified )
 
     tableNames.push_back( tableName );
   }
-  if ( rc != SQLITE_DONE )
+  if ( rc != SQLITE_DONE && rc != SQLITE_ROW )
```

Comment:
why do we need to handle SQLITE_ROW here as well? (and in other places in this file)

Reply:
Looks like the last time we call `sqlite3_step()` assigment does not work and we get following message in logs
```
SQLITE3 error [100]: another row available
```

Reply:
Should be fixed now. The proble was with geometry columns retrieval, we call `sqlite3_step()` only once, so it does not reach `SQLITE_DONE` state.